src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -55,18 +55,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String username = request.getParameter(USERNAME_PROPERTY_KEY);
     String bio = request.getParameter(BIO_PROPERTY_KEY);
     boolean edit = Boolean.parseBoolean(request.getParameter(EDIT_PROPERTY_KEY));
-
+    
+    
     if (!areValidParameters(name, username, bio)) {
       response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Post Request parameters empty");
       return;
     }
     if (edit) {
-      profileData.editProfile(name, username, bio);
+      if (profileData.doesUsernameExist(username)) {
+        PrintWriter out = response.getWriter();  
+        response.sendRedirect("/EditProfilePage.html");
```

Comment:
Is there some sort of notification for the user to know why they got redirected?

Reply:
I'm still working on having some sort of an alert to the user